src/core/agent/services/signifyNotificationService.ts

```diff
@@ -223,22 +231,15 @@ class SignifyNotificationService extends AgentService {
             exchange.exn.i,
           ]);
         await Agent.agent.ipexCommunications.createLinkedIpexMessageRecord(
-          existingCredential.schema.title,
-          exchange.exn.i,
           exchange,
-          ConnectionHistoryType.CREDENTIAL_UPDATE
+          ConnectionHistoryType.CREDENTIAL_ISSUANCE
         );
         await this.markNotification(notif.i);
         return;
       } else {
-        const schema = await this.props.signifyClient
-          .schemas()
-          .get(exchange.exn.e.acdc.s);
         await Agent.agent.ipexCommunications.createLinkedIpexMessageRecord(
-          schema.title,
-          exchange.exn.i,
           exchange,
-          ConnectionHistoryType.CREDENTIAL_ISSUANCE
+          ConnectionHistoryType.CREDENTIAL_UPDATE
```

Comment:
I'm not sure why these got swapped. If the credential already exists, it's an update. If not, it's an issuance